adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -419,6 +421,13 @@ - (NSString *)getSuperToken {
   return [VungleSDK.sharedSDK currentSuperToken];
 }
 
+- (void)setCOPPAStatus:(NSNumber *)coppa {
+    if (coppa) {
+        [[VungleSDK sharedSDK] updateCOPPAStatus:[coppa boolValue]];
```

Comment:
Seems like the publishers don't need to use another Vungle API to set the COPPA status, but the adapter can just get what's set through the Google Mobile Ads SDK and pass it to Vungle SDK. Is that correct?

Reference:
- https://developers.google.com/admob/ios/targeting#child-directed_setting
- https://developers.google.com/admob/ios/api/reference/Classes/GADMediationAdConfiguration?hl=en#childdirectedtreatment

Reply:
Yes. That's correct. The publisher never uses the Router class. The Router class is a wrapper to communicate with the Vungle SDK. I'm following the implementation guidelines for this adapter with this function.